src/main/java/ReadFileAndPrintIt.java

```diff
@@ -0,0 +1,40 @@
+package main.java;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+
+public class ReadFileAndPrintIt {
+    public static void main(String[] args) {
+
+        String directoryPath = "test";
+        String currentDirectory = System.getProperty("user.dir");
+
+        File directory = new File(currentDirectory, directoryPath);
+
+        String fileName = "file.txt";
+
+        File file = new File(directory, fileName);
+
+        if (file.exists()) {
+            try {
+                BufferedReader bufferedReader = new BufferedReader(new FileReader(file));
+                String string;
+                while ((string = bufferedReader.readLine()) != null) {
+                    System.out.println(string);
+                }
+
+                bufferedReader.close();
```

Comment:
How to automatically deallocate the file descriptor.
Check 'try with resources' topic